seedcase_sprout/core/sprout_checks/data/data_type_checks.py

```diff
@@ -0,0 +1,94 @@
+import json
+import re
+import uuid
+
+import xmlschema
+
+# Data types for validating time and date values
+XML_SCHEMA_TYPES = xmlschema.XMLSchema11(
+    '<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"/>'
+).types
+
+
+def check_xml_type(value: str, xml_type: str) -> bool:
+    """Checks if the given `value` is formatted as a valid XML data type.
+
+    The Frictionless Data Package standard follows the definitions at
+    https://www.w3.org/TR/xmlschema-2/ for time and date-related data types.
+    This function is for checking values against these XML data type definitions.
+
+    Args:
+        value: The value to check.
+        xml_type: The XML data type to check against.
```

Comment:
```suggestion
        type: The XML data type to check against.
```

Since `xml_type` is already in the name of the function.